dodo.py

```diff
@@ -0,0 +1,29 @@
+def task_build_docker_image():
+    return {
+        "actions": [
+            "docker build -f metaflow/plugins/kfp/tests/Dockerfile -t metaflow-integration-testing:1.0 ."
+        ],
+    }
+
+
+# TODO AIP-1766 Move compile_pipeline task to aip-doit-modules
+def task_run_integration_tests():
+    """Run the full integration suite inside a Docker container."""
+    return {
+        "actions": [
+            "docker run --rm "
+            + "-e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID "
+            + "-e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY "
+            + "-e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN "
+            + "-e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION "
```

Comment:
does AlexL mount the `/.aws` or does he create environment variables?

Reply:
Alex suggested I don't mount for now, as the AWS keys are exported and not put into a `~/.aws` directory.